packages/api/amplify_api/lib/src/api_plugin_impl.dart

```diff
@@ -116,7 +115,7 @@ class AmplifyAPIDart extends AmplifyAPI {
   ///
   /// Use [apiName] if there are multiple endpoints of the same type.
   @visibleForTesting
-  http.Client getHttpClient(EndpointType type, {String? apiName}) {
+  AWSHttpClient getHttpClient(EndpointType type, {String? apiName}) {
```

Comment:
I was getting a lot of errors in android w http, so would suggest we make the http clients set to http/1 for now. 

This got it working for me:

```
return _clientPool[endpoint.name] ??= AmplifyHttpClient(
      baseClient: AmplifyAuthorizationRestClient(
        endpointConfig: endpoint.config,
        baseClient: _baseHttpClient,
        authProviderRepo: _authProviderRepo,
      ),
    )..supportedProtocols = SupportedProtocols.http1;
```